app/controllers/shf_applications_controller.rb

```diff
@@ -140,6 +138,7 @@ def create
         render :edit
       end
 
+      send_new_app_emails(@shf_application)
```

Comment:
Since the preceding if-else statement does not execute `return` after `redirect` or `render`, won't this statement execute regardless of the outcome of the if-else statement?

Reply:
Yup.  By design. 
The mailer checks to see if there are or are not attached to the application and puts in the appropriate wording.